changelog/changes/2023-10-17-sdk-mantle-kola-sysext.md

```diff
@@ -0,0 +1 @@
+- SDK: bump Mantle / Kola container image version to include torcx removal / sysext tests ([Mantle#466](https://github.com/flatcar/mantle/pull/466), [Mantle#465](https://github.com/flatcar/mantle/pull/465))
```

Comment:
This entry file can be removed because it's rather something for the git log and not for the user-facing release notes

Reply:
Gotcha, will update.